### PR TITLE
Min learning rate ratio

### PR DESCRIPTION
**Pull request by psinger**
Adds a new setting `min_learning_rate_ratio` that allows to specify a minimum ratio for cosine and linear schedulers.

Closes https://github.com/h2oai/h2o-llmstudio/issues/671

### CODE REVIEW

File: documentation/docs/tooltips/experiments/_min-learning-rate-ratio.mdx
Comment (pascal-pfeiffer) [inline documentation/docs/tooltips/experiments/_min-learning-rate-ratio.mdx:1]: Can you please also add that to the documentation.

File: llm_studio/python_configs/text_causal_language_modeling_config.py [hunk 1]
```diff
@@ -211,6 +212,7 @@ def __post_init__(self):
         self._possible_values["batch_size"] = (1, 256, 1)
         self._possible_values["epochs"] = (0, 10, 1)
         self._possible_values["schedule"] = Schedulers.names()
+        self._possible_values["min_learning_rate_ratio"] = (0.0, 0.1, 0.0001)
```
Comment (pascal-pfeiffer) [inline llm_studio/python_configs/text_causal_language_modeling_config.py:215]: As we limit it here to 0.1, but say it can go to 1.0 in the tooltip, shall we make it customizable in the settings?

**Review by pascal-pfeiffer — CHANGES_REQUESTED** [review]
Thank you, works well. Just the tooltip should be added/imported to the documentation before merged.

File: llm_studio/python_configs/text_causal_language_modeling_config.py (hunk 1 above)
Comment (psinger) [reply llm_studio/python_configs/text_causal_language_modeling_config.py:215]: will adjust the tooltip

**Review by pascal-pfeiffer — APPROVED** [review]
lgtm
